Corvus.Extensions.Samples/TraversalExtensionsSample.dib

```diff
@@ -0,0 +1,248 @@
+#!markdown
+
+## Traversal Extensions - Sample Notebook
+
+#!markdown
+
+First, add a reference to the `Corvus.Extensions` NuGet package.
+
+#!csharp
+
+#r "nuget: Corvus.Extensions, 1.1.4"
+
+using Corvus.Extensions;
+
+#!markdown
+
+### ForEachAsync()
+
+#!markdown
+
+Execute an async action for each item in the enumerable.
+
+#!csharp
+
+public async Task<bool> isEqualToOne(int x)
+{
+    bool result = x.Equals(1);
+    Console.WriteLine(result);
+    return result;
+}
+
+Func<int, Task<bool>> funcIsEqualToOne = isEqualToOne;
+
+#!csharp
+
+var list = new List<int> { 1, 2, 3, 4, 5 };
+await list.ForEachAsync<int>(isEqualToOne)
+// method was useful when we didn't have async and await.
```

Comment:
I don't understand this comment - is it still relevant?

Reply:
I've changed the comment and added it in a cell in markdown.
When we talked to Ian about `ForEachAsync()`, we concluded the method was no longer useful nowadays as the same output can be achieved with a `foreach` loop.

Reply:
We should probably think about removing that method....